### ISSUE TITLE
Can't preview the charts

### ISSUE THREAD

**Issue by lg2578** [AppPear/ChartView#66]
I‘ve forked the swiftuicharts,and cloned by xcode.
but the charts can't be previewed.
for example,I  select the multiLineChartView ,and click the preview 
nothing display in the canvas

**Comment by shah95rukh** [comment]
I just cloned it yesterday and it works for me. Are you using it in SwiftUI?

**Comment by RamonBrisk** [comment]
yeah, the same ,no lines inside the chart, just the chart frame shows.
